src/components/common/customerService/CustomerServiceHeader.tsx

```
@@ -0,0 +1,32 @@
+import { MagnifyingGlassIcon } from '@heroicons/react/24/outline';
+import * as S from './CustomerServiceHeader.styled';
+import { ROUTES } from '../../../constants/routes';
```

ESLint

[error] 'ROUTES' is defined but never used.

(@typescript-eslint/no-unused-vars)